src/components/TransactionFlow/TxReceipt.tsx

```diff
@@ -160,7 +167,17 @@ const TxReceipt = ({
       const provider = new ProviderHandler(txConfig.network);
       const timestampInterval = setInterval(() => {
         getTimestampFromBlockNum(blockNumber, provider).then((transactionTimestamp) => {
-          if (sender.account && !disableAddTxToAccount) {
+          if (txReceipt && txReceipt.txType === ITxType.FAUCET) {
+            const recipientAccount = getStoreAccount(accounts)(txReceipt.to, txConfig.network.id);
```

Comment:
Can this ever be undefined?

Reply:
Types for the component props are defined in [src/types/transactionFlow.ts](https://github.com/MyCryptoHQ/MyCrypto/blob/afbed58548c6ffc60ebd4a7c2f3b6f65e06f319f/src/types/transactionFlow.ts#L87), it's needed because of that type definition.

```typescript
export interface IStepComponentProps extends IDefaultStepComponentProps {
  [...]
  txReceipt?: ITxReceipt;
  [...]
}
```

In reality the Faucet component will always pass a valid txReceipt prop to the TxReceipt component, not sure if any other components in the codebase ever pass `undefined` though.